ui-kit/src/main/kotlin/org/p2p/uikit/organisms/seedphrase/adapter/SeedPhraseAdapter.kt

```diff
@@ -46,12 +46,12 @@ class SeedPhraseAdapter(
         super.onViewRecycled(holder)
     }
 
-    fun replaceSecretKeys(seedPhrase: List<SeedPhraseWord>) {
+    fun replaceSeedPhraseItems(seedPhrase: List<SeedPhraseWord>) {
         clear()
         addAllSecretKeys(seedPhrase)
     }
 
-    fun addSecretKey(seedPhraseWord: SeedPhraseWord) {
+    fun addSeedPhraseWordItem(seedPhraseWord: SeedPhraseWord) {
```

Comment:
Just `addWord`
The adapter name is SeedPhrase.
It's obvious that we are adding the word into seed phrase